register.php

```diff
@@ -27,16 +27,16 @@
 			<div class="col text-center">
 				<img src="logo.png" class="my-4" alt="Stravan't">
 				<h1>Register a new account</h1>
-				<form method="POST">
+				<form method="POST" action="create-account.php">
```

Comment:
I had the registration form submit to the same page so that it would be easy to display errors (e.g. passwords don't match, invalid email)